test/api/test_consent_type.py

```diff
@@ -0,0 +1,138 @@
+from ..utils import Endpoint
+from fastapi import HTTPException
+
+endpoint = Endpoint()
+
+
+def test_post_consent_type():
```

Comment:
THis test adds consent but does not remove it at the end. Next time you run this test it may fail as consent already exists. If someone runs this test on the production env it may corrupt its data. Always clean after tests.